.gitignore

```diff
@@ -0,0 +1,2 @@
+
+node_modules
```

Comment:
- [ ] Kindly add a test.md in your .gitignore as requested in the requirements.
> There is a .gitignore file listing test.md file and node_modules directory (as it can be added by accident while setting up Linters).